TwitterTweetSentimentalAnalysis/Model.py

```diff
@@ -0,0 +1,87 @@
+# this is a local Credential Helper
+import tweepy
+from nltk.sentiment import SentimentIntensityAnalyzer
+from textblob import TextBlob
+
+import TwitterTweetSentimentalAnalysis.CredentialHelper as twitterCredential
+
+
+# Tweepy is a lib that can be used to connect to twitter api
```

Comment:
remove this comment and make sure that the space between the last import statement and class statement below is not more than 2 lines